website/pages/docs/glossary/what-is-the-modern-data-stack.mdx

```diff
@@ -0,0 +1,97 @@
+
+## What is a “Stack”?
+
+A stack is a collection of technologies that combine to form a system.
+Usually, components of the stack can be swapped out for others.
+The modern data stack typically includes the following base components: **Data Sources**, **ELT**, **Data Storage**, **Data Transformation**, **Orchestration**, **Monitoring**, **Governance** and **Data Analysis and Visualization**.
+Though as with any engineering stack you can plug and play different tools at each component and use only some of the components, depending on your needs and use cases.
+
+## How did the Modern Data Stack Come About?
+
+Prior to what is now called the modern data stack we had standard Relational Database Management System (RDBMS) used for analytical workloads which made it hard to scale to big data due to its storage/compute coupling and OLTP architecture.
+
+The modern data stack started gaining popularity in the early 2010s, enabled by cloud computing and OLAP data warehouse and data lake technologies such as Redshift, BigQuery, Snowflake, and more recent innovations like ClickHouse and in-memory OLAP technologies like DuckDB. Even though it has been around for a while, the popularity of this approach has grown significantly, and we are seeing a lot of innovation in every part of the stack.
+
+The introduction of data warehouses and data lakes also made it possible to shift Extract-Transform-Load (ELT) workloads to Extract-Load-Transform (ELT). This approach moves transformation to the data warehouse and data lake which gives much more flexibility to teams working with the data without the need to ingest the data again and running compute intensive operations.
+
+# Architecture
+
+![The Modern Data Stack Architecture](/images/blog/what_is_the_modern_data_stack/architecture.png)
+
+**Note:** There are many more vendors at each component of the stack and we chose some of them in no particular order (apart from CloudQuery given our bias :)).
+
+## Data Sources
+
+This refers to the data we want to ingest and involves different types of sources:
+
+- **Databases**: Companies have many databases for different workloads. Since these databases are not optimized for analytical workloads, it is often necessary to ingest data from production databases to analytical databases where you can run complex queries without impacting the production system. Additionally, there have been many advances around Change Data Capture ([CDC](https://www.cloudquery.io/blog/postgres-cdc-to-any-destination)) for databases, allowing you to ingest data in real-time from databases. This enables efficient, near-real time syncing between source databases and destinations.
```

Comment:
```suggestion
- **Databases**: Companies have many databases for different workloads. Since these databases are not optimized for analytical workloads, it is often necessary to ingest data from production databases to analytical databases where you can run complex queries without impacting the production system. Additionally, there have been many advances around Change Data Capture ([CDC](/blog/postgres-cdc-to-any-destination)) for databases, allowing you to ingest data in real-time from databases. This enables efficient, near-real time syncing between source databases and destinations.
```